pkg/apis/v1alpha1/dnspolicy_types.go

```diff
@@ -33,6 +33,36 @@ type DNSPolicySpec struct {
 
 	// +optional
 	HealthCheck *HealthCheckSpec `json:"healthCheck,omitempty"`
+
+	// +kubebuilder:validation:Required
+	// +required
+	LoadBalancing *LoadBalancingSpec `json:"loadBalancing"`
+}
+
+type LoadBalancingSpec struct {
+	// +required
+	Weighted *LoadBalancingWeighted `json:"weighted,omitempty"`
+	// +optional
+	Geo *LoadBalancingGeo `json:"geo,omitempty"`
+}
+
+type LoadBalancingWeighted struct {
+	// default is the record weight to use when no other can be determined for a dns target cluster.
+	//
+	// +kubebuilder:default=120
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=120
+	Default int `json:"default,omitempty"`
+}
+
+type LoadBalancingGeo struct {
+	// default is the country or continent code to use when no other can be determined for a dns target cluster.
+	//
+	// 2 digit iso3166 country code (https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) or continent code:
+	// AF: Africa; AN: Antarctica; AS: Asia; EU: Europe; OC: Oceania; NA: North America; SA: South America
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=2
+	Default string `json:"default,omitempty"`
```

Comment:
@maleck13 Based on yesterdays call do we want to rename these? Looking at the [docs](https://gateway-api.sigs.k8s.io/references/policy-attachment/#inherited-policy-attachment-its-all-about-the-defaults-and-overrides) for the inherited policy attachment i can see where confusion might come in, and maybe just changing it now would be a good idea. 

Maybe `defaultWeight` and `defaultGeo`? It is a little more obvious what it is this way, `default` is not the most descriptive.

/cc @thomasmaas @guicassolato

Reply:
yeah I think changing them now is reasonable. There could come a point where we want to use inherited rather than direct policy attachment